src/styles/homepage.css

```diff
@@ -0,0 +1,74 @@
+.shows {
+  width: 100%;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 45px 30px;
+  padding: 20px 0;
+  background: #242222;
+}
+
+.show {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
```

Comment:
[OPTIONAL] Kindly you can put the name and likes in the margin of each image container.